src/components/createAppointment/doctorPicker/doctorPicker.styles.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components'
+import Select from 'react-select'
+
+const Form = styled.form``
+
+const StyledSelect = styled(Select)`
+  & .Select {
+    &__value-container {
+      padding: 8px 24px;
+      font-weight: 400;
+      font-size: 17px;
+      line-height: 1.4;
+    }
+
+    &__single-value {
+      color: var(--title-color);
+    }
+
+    &__placeholder {
+      color: var(--text-color);
+    }
+
+    &__menu {
+      overflow-y: scroll;
```

Comment:
I think you can simply set overflow-y: hidden

Reply:
fixed 
thx mate